Update MAC_SETUP.md (#146)
ONSdigital/dp: guides/MAC_SETUP.md

Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.

### Testing SSH Connection

> To check that you've setup your Git SSH on your terminal open your favoured terminal and enter:

Reviewer: - setup -> set up
(noun -> verb)
- comma 

```suggestion
To check that you've set up your Git SSH on your terminal, open your favoured terminal and enter:
```

  * `ssh -T git@github.com`
  

> You will then be asked to type *yes* if you're ready to connect.  You should then be then be prompted with:

Reviewer: - Double `then be` we can remove one
- Removal of double spaces (think the rest of our document uses single space after full-stop so just a consistency thing although optional)
```suggestion
You will then be asked to type *yes* if you're ready to connect. You should then be prompted with:
```

  * `Hi [YOUR GIT USERNAME]! You've successfully authenticated, but GitHub does not provide shell access.`

> You can then close your terminal.  If its unsuccessful check you've used the correct email address (email used for Github account) when creating and saving your key.

Reviewer: - Removal of double spaces (think the rest of our document uses single space after full-stop so just a consistency thing although optional)
- Added apostrophe to `its`
```suggestion
You can then close your terminal. If it's unsuccessful check you've used the correct email address (email used for Github account) when creating and saving your key.
```
